server/src/main/resources/application.example.yml

```diff
@@ -194,6 +194,22 @@ seata:
       vgroup-table: vgroup_table
       query-limit: 1000
       max-wait: 5000
+      druid:
+        timeBetweenEvictionRunsMillis: 120000
```

Comment:
In general, YAML files commonly use lowercase letters and hyphens to separate different words. This is the recommended `Kebab Case` style for Spring configurations.

Reply:
Yes.I will correct it.

Reply:
> In general, YAML files commonly use lowercase letters and hyphens to separate different words. This is the recommended `Kebab Case` style for Spring configurations.

Yes.I will correct it.

Reply:
> In general, YAML files commonly use lowercase letters and hyphens to separate different words. This is the recommended `Kebab Case` style for Spring configurations.

It's done.